src/test/java/nl/praegus/fitnesse/responders/testHistory/TestHistoryRecentTest.java

```diff
@@ -39,15 +39,17 @@ public void When_directory_is_not_null_return_historylines_not_sorted() {
     }
 
     @Test
-    public void When_dateTimeFormatter_is_not_working_correctly(){
+    public void checks_date_is_correct() throws ParseException {
```

Comment:
wanneer is de date correct? De naam van deze test maakt nog niet heel duidellijk wat je nu eigenlijk test. De formatting van de datum?

Reply:
De naam van de test is veranderd, zodat het de test beter omschrijft.